### PR TITLE
Add missing styles on empty state SEO & Traffic page

### PR DESCRIPTION
**Pull request by CaptainYouz**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.5.x
| Description?  | This PR add the missing styles in empty states SEO.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | #10683 
| How to test?  | Check before after

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10718)

### CODE REVIEW

File: admin-dev/themes/new-theme/scss/components/_showcase-card.scss [hunk 1]
```diff
@@ -16,7 +16,10 @@
 
   &__left {
     width: 320px;
+    animation: showcase-img-appereance 1s;
+
     img {
+      animation: showcase-img-appereance 1.5s;
```
Comment (matks) [inline admin-dev/themes/new-theme/scss/components/_showcase-card.scss]: `appereance` => `appearance`

**Review by matks — DISMISSED** [review]

**Comment by Quetzacoalt91** [comment]
@CaptainYouz I guess this PR should be on `1.7.5.x` branch?

**Comment by Quetzacoalt91** [comment]
@matks, your comments have been applied.

PR rebased on 1.7.5.x and conflicts resolved

**Review by Quetzacoalt91 — DISMISSED** [review]

**Comment by Quetzacoalt91** [comment]
Rebased

**Review by Quetzacoalt91 — APPROVED** [review]

**Pull request merged**